contracts/duckies/games/DuckyFamily/DuckyFamilyV1.sol

```diff
@@ -467,49 +472,50 @@ contract DuckyFamilyV1 is IDuckyFamily, AccessControl, Random {
 	 * @param collectionId Collection ID.
 	 * @return genome Generated genome.
 	 */
-	function _generateGenome(uint8 collectionId) internal returns (uint256) {
+	function _generateGenome(uint8 collectionId) internal view returns (uint256) {
 		if (collectionId != ducklingCollectionId && collectionId != zombeakCollectionId) {
 			revert MintingRulesViolated(collectionId, 1);
 		}
 
+		(bytes3 seedChunk, bytes32 seed) = _rotateSeedChunk(_randomSeed());
+
```

Comment:
Why just not making a method:
```
getRandom() returns uint24
```

How do you know the chunk number ?

Reply:
My point is that is a random number but not a seed

Reply:
I am still for naming this data as `seed`, because it takes part in random number generation